04-python/solutions/adults_children.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+# Import the users list from `data.py`:
+import data
+
+
+def group_by_age(users):
+    result = {"adults": [], "children": []}
+    for user in users:
+        if user["age"] < 18:
+            result["children"].append(user)
+        else:
+            result["adults"].append(user)
+    return result
+
+
+# See https://docs.python.org/3/library/json.html
+print(json.dumps(group_by_age(data.users), indent=4))
```

Comment:
Maybe https://docs.python.org/3/library/pprint.html would be easier to understand since this exercise doen't really have something to do with json. But it doesn't look as pretty:
```
{'adults': [{'age': 54, 'first_name': 'Brain', 'last_name': 'Mohr'},
            {'age': 28, 'first_name': 'Franz', 'last_name': 'Raynor'},
            {'age': 61, 'first_name': 'Celestino', 'last_name': 'Bailey'}],
 'children': [{'age': 17, 'first_name': 'Bella', 'last_name': 'VonRueden'}]}
```